src/lm_buddy/jobs/ragas/entrypoint.py

```diff
@@ -0,0 +1,110 @@
+import os
```

Comment:
i wonder if we should have /jobs/eval/ragas...etc since we will have more than 1 eval type as a structure